jekyll/_docs/build-image-trusty.md

```diff
@@ -3,11 +3,14 @@ layout: classic-docs
 title: Ubuntu 14.04 (Trusty)
 categories: [build-images]
 description: Ubuntu 14.04 (Trusty)
+changefreq: "weekly"
```

Comment:
This is a signal to the generated sitemap (sitemap.xml for Google) on approximately how often the page might be updated. This in turn hints to search engines like Google how often their search spiders should come back and check the page.